### PR TITLE
#1: bootstrapping package.json 

### PR DESCRIPTION
**Pull request by KorolevskiTigr2**
#1

### CODE REVIEW

**Comment by gnrlbzik** [comment]
@KorolevskiTigr2 good job on creating your first PR

File: package.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "name" : "project"
```
Comment (gnrlbzik) [inline package.json:2]: nit: please update the name of the project.

Comment (gnrlbzik) [inline package.json:2]:
```suggestion
{
  "name": "korolevski-chasi",
  "version": "0.0.0",
  "description": "A time conversion project",
  "main": "",
  "private": true,
  "scripts": {
    "test": "",
  },
  "dependencies": {},
  "devDependencies": {},
  "engines": {
    "node": "16.15.1",
    "npm": "8.11.0"
  }
}

```

Comment (KorolevskiTigr2) [reply package.json]: done

**Review by gnrlbzik — APPROVED** [review]
excellent first step, good job

**Pull request merged**